### ISSUE TITLE
Change NPM main file

### ISSUE THREAD

**Issue by mistic100** [tests-always-included/angular-placeholder#17]
It would be interresting that the angular modules export its name for easy use in webpack.

I would be as simple as creating a index.js file with 
```js
require('./lib/tai-placeholder');
module.exports = 'taiPlaceholder';
``` 
and use this as main in package.json

The difference is the usage of module.exports with the name of the ng module.

See for example what the angular-animate package looks like.

**Comment by mistic100** [comment]
The point is being able to do 

```js
angular.module('myApp', [
  require('angular-placeholder')
]);
```

**Comment by fidian** [comment]
Is this a pattern that Angular libraries typically follow? Is there Webpack documentation on this approach?

I ask because I don't use Webpack and this works great with Browserify as-is. Also, if I change the main entry to `index.js` with the content you suggested, none of the code would be exported.

Wouldn't the exact same effect be accomplished if I had `module.exports = "taiPlaceholder";` as the last line in `lib/tai-placeholder.js`?

**Comment by mistic100** [comment]
Or seems that All angular official modules follow this pattern.

The code actually does not need to be exported, out just needs to be executed to register the module inside angular.

You can indeed make the change in the current main file but don't forget to check that module.exports actually exists (for bower and "old-school" users)

**Comment by mistic100** [comment]
On the other hand I don't know Browserify will react the the change. But if you already use things like angular-animate or angular-cookies it shouldn't be a problem.

**Comment by fidian** [comment]
This appears to have no problems whatsoever for me and it helps you out. I've added a test to confirm the functionality and everything appears to be working as expected. I've released 1.1.0 in the hopes that it works for you. Let me know if I didn't do it right.

**Comment by mistic100** [comment]
Thank you, works perfectly.
